.github/workflows/master-build-push.yaml

```diff
@@ -1,4 +1,4 @@
-name: Push Image Build
+name: Push Master Image Build
```

Comment:
no particular reason, I wanted to keep it clean for some time so that in case something fails it will direct things to look. But right we can remove duplicacy.